container/default.json

```diff
@@ -128,21 +128,21 @@
   "footer": {
     "links": {
       "text": "<0>ISIS Home</0> | <2>Privacy statement</2> | <4>Data policy</4> | <6>Accessibility statement</6> | <8>Contact</8>",
-      "facility": "#",
-      "privacy-statement": "#",
-      "data-policy": "#",
-      "contact": "#"
+      "facility": "https://www.isis.stfc.ac.uk/Pages/home.aspx",
+      "privacy-statement": "/cookies",
```

Comment:
Does this exist right now? if so this is fine

Reply:
The cookies page? Yeah though have just kept as it is from SciGateway